src/lib/zAuction.ts

```diff
@@ -87,6 +87,39 @@ export async function getBidsForNft(
 	return bids;
 }
 
+export async function getBidsListForNfts(
```

Comment:
Think about what is going to be easiest for a user to use.

In our application is there any case where a user is going to want to get bids from multiple NFT's from multiple contracts? We only are ever dealing with one contract per network, but your interface asks for an array of contracts.

This makes it cumbersome for a user to use since if they want to get bids from 100 different tokens, they're going to have to fill a second array with 100 of the same value.

I recommend you make the contract input a `contract: string` and just re-use the contract.